components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -763,7 +763,7 @@
             if (response.statusText == '' /* No response */ || response.status == 12031 /* In IE */) {
                failureReason = "$services.localization.render('phenotips.patientAccessRightsManagement.modifyNoServerResponse')";
             }
-            _this._messages.update(new Element('div', {'class' : 'errormessage'}).update("$services.localization.render('phenotips.patientAccessRightsManagement.modifyUpdateFailure')" + failureReason));
+            _this._messages.update(new Element('div', {'class' : 'errormessage'}).update("$services.localization.render('phenotips.patientAccessRightsManagement.modifyUpdateFailure')" + " " + failureReason));
```

Comment:
You can just leave a trailing space inside the quote before: `....modifyUpdateFailure') " + failureReason));`